### DISCUSSION TITLE
Mise à jour 3.4.3b vers 3.5.0 - Parse error

### DISCUSSION THREAD

**Discussion by acdct** [JeromeDevome/GRR#238] [Q&A] [answered]
Bonjour,

Je suis actuellement en version 3.4.3b de GRR.

Je tente de mettre à jour vers la version 3.5.0 et je tombe toujours sur le message :
`Parse error: syntax error, unexpected ':', expecting '{' in C:\xampp\htdocs\grr\include\language.inc.php on line 190`

Je suis en PHP 5.6.40, dans les prérequis il faut au moins la 5.6.0 pour pouvoir passer en version 3.5.0.

Merci d'avance pour votre aide.

Cordialement

**Comment by ynaessens** [comment, marked as answer]
Bonjour,
je viens de tester avec php 5.6.40 et effectivement il y a un souci de syntaxe.
Si vous validez les modifications faites récemment en récupérant les fichiers de GRR à partir de cette adresse : https://github.com/JeromeDevome/GRR/tree/GRR-3.5.0
je publierai une version corrective.
Cordialement,
YN

**Reply by acdct** [reply]
Bonjour,

Je vous confirme que les fichiers récupérés à l'adresse indiquée me permettent de faire la mise à jour sans problème.

Pouvez-vous aussi me confirmer que la dernière version installable pour un PHP 5.6.40 sera la 3.5.1 dès qu'elle sera publiée en stable ?

Merci beaucoup pour votre aide et votre réactivité !

Bonne journée.
